server.py

```
@@ -369,6 +706,66 @@ async def _api_chat(req: ChatRequest):
+        return {
+            "config": config_to_dict(_graph_config),
+            "soul": read_soul(),
+        }
+
+    @fastapi_app.post("/api/config")
+    async def _api_post_config(req: ConfigReloadRequest):
+        ok, messages = _apply_settings_changes(config=req.config, soul=req.soul)
+        return {"ok": ok, "messages": messages}
+
+    @fastapi_app.get("/api/config/models")
+    async def _api_list_models(api_base: str = "", api_key: str = ""):
```

Ruff (0.15.10)

[warning] Missing return type annotation for private function `_api_list_models`

(ANN202)